requirements-dev.txt

```diff
@@ -1,8 +1,11 @@
+-r requirements.txt
+
 black~=23.7.0
 isort~=5.12.0
 matplotlib>=3.5.1
```

Comment:
I think you can remove `matplotlib` from the dependencies + the import that's now useless inside the file `test_duration_generator.py`